docs/contributing/development-policies.rst

```diff
@@ -0,0 +1,219 @@
+can deploy this version confident that keeping it up-to-date requires only easily-applied security and other critical
+updates, until the next LTS release.
+
+
+.. _branch_policy:
+
+********
+Branches
+********
+
+..  versionchanged:: 3.3
+
+    Previously, we maintained a ``master`` branch (now deleted), and a set of ``support`` branches (now pruned, and
+    renamed ``release``).
+
+We maintain a number of branches on `our GitHub repository <https://github.com/divio/django-cms>`_.
+
+the latest (highest-numbered) ``release/x.y.z``
+    This is the branch that will become the next release on PyPI.
+
+    **Fixes and backwards-compatible improvements** (i.e. most pull requests) will be made against
+    this branch.
+
+``develop``
+    This is the branch that will become the next release that increments the ``x`` or ``y`` of the latest
+    ``release/x.y.z``.
+
+    This branch is for **new features and backwards-incompatible changes**. By their nature, these will require more
+    substantial team co-ordination.
+
+Older ``release/x.y.z`` branches
+     These represent the final point of development (the highest ``y`` of older versions). Releases in the full set of
+     older versions have been tagged (use Git Tags to retrieve them).
+
+     These branches will only rarely be patched, with security fixes representing the main reason for a patch.
+
+Commits in ``release/x.y.z`` will be merged forward into ``develop`` periodically by the core developers.
+
+If in doubt about which branch to work from, ask on the #django-cms IRC channel on `freenode`_ or the
+`django-cms-developers`_ email list!
+
+
+.. _commit_policy:
+
+*******
+Commits
+*******
+
+.. versionadded:: 3.3
+
+Commit messages
+===============
+
+Commit messages and their subject lines should be written in the past tense, not present tense, for example:
+
+    Updated contribution policies.
+
+    * Updated branch policy to clarify purpose of develop/release branches
+    * Added commit policy.
+    * Added changelog policy.
+
+Keep lines short, and within 72 characters as far as possible.
+
+
+Squashing commits
+=================
+
+In order to make our Git history more useful, and to make life easier for the core developers, please rebase and
+squash your commit history into a single commit representing a single coherent piece of work.
+
+For example, we don't really need or want a commit history, for what ought to be a single commit, that looks like
+(newest last)::
+
+    2dceb83 Updated contribution policies.
+    ffe5f2c Fixed spelling mistake in contribution policies.
+    29168da Fixed typo.
+    85d925c Updated commit policy based on feedback.
+
+The bottom three commits are just noise. They don't represent development of the code base. The four commits
+should be squashed into a single, meaningful, commit::
+
+    85d925c Updated contribution policies.
+
+
+How to squash commits
+---------------------
```

Comment:
Perhaps we could state that "GitHub does this for us, but if you prefer to do it manually..."